pages/media/index.tsx

```diff
@@ -26,43 +26,25 @@ export default function Media( props: Props ) {
 						const {
 							altText = '',
 							id,
-							mediaDetails: {
-								height,
-								width,
-							},
 							sourceUrl,
 						} = mediaItem;
 
 						// Each image is displayed in a fixed-height box of 100px. If the
 						// actual height of the image is less than 100px, then use
-						// layout=fill and objectFit to fill the box.
 						return (
 							<a
-								className={styles.image}
+								className={styles['image-link']}
 								href={sourceUrl}
 								key={id}
 								rel="noreferrer"
 								target="_blank"
 							>
-								{
-									height < 100
-										?
-										<Image
-											alt={altText}
-											layout="fill"
-											objectFit="cover"
-											objectPosition="0"
-											src={sourceUrl}
-										/>
-										:
-										<Image
-											alt={altText}
-											layout="intrinsic"
-											height={100}
-											src={sourceUrl}
-											width={width / height * 100}
-										/>
-								}
+								<Image
+									alt={altText}
+									className={styles.image}
+									fill={true}
```

Comment:
Just a comment, did you test if it works using `fill` property without `true` value?
`<Image fill {...} />`

Reply:
Seems like that'd work from [the code example in NextJS's documentation](https://nextjs.org/docs/api-reference/next/image#sizes):

> ```jsx
> import Image from 'next/image'
> const Example = () => (
>  <div className="grid-element">
>    <Image
>      src="/example.png"
>      fill
>      sizes="(max-width: 768px) 100vw,
>              (max-width: 1200px) 50vw,
>              33vw"
>    />
>  </div>
> )
> ```